python/batch-predictor/merlinpyspark/__main__.py

```diff
@@ -80,10 +83,18 @@ def local_spark_session():
     )
     conf.set(
         "spark.jars.packages",
-        "com.google.cloud.spark:spark-bigquery-with-dependencies_2.11:0.13.1-beta",
+        "com.google.cloud.spark:spark-bigquery-with-dependencies_2.12:0.13.1-beta",
+    )
+
+    conf.set(
+        "spark.jars",
+        "https://github.com/caraml-dev/caraml-store/releases/download/caraml-store-spark/odps/v0.0.1-test/custom-dialect.jar,"
+        "https://github.com/aliyun/aliyun-odps-jdbc/releases/download/v3.8.2/odps-jdbc-3.8.2-jar-with-dependencies.jar,"
+        "https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-hadoop2-2.0.1.jar"
     )
 
     sc = SparkContext(conf=conf)
+    sc.setLogLevel("INFO")
```

Comment:
Why do we need to change the log level though?

Reply:
Just for logging purposes, default is warning which doesn't give much information

Reply:
I see.. alright then I was just wondering why there's a change. Thanks!